src/main/resources/application.yaml

```diff
@@ -97,7 +97,7 @@ spring:
         registration:
           oidc:
             client-id: microservicekey-am-role-assignment-service
-            client-secret: ${AM_ROLE_ASSIGNMENT_SERVICE_SECRET:AAAAAAAAAAAAAAAA}
+            client-secret: ${AM_ROLE_ASSIGNMENT_SERVICE_SECRET:}
```

Comment:
you also need to remove default value from below statement. Ideally, It should fetch from vault.
username: ${ROLE_ASSIGNMENT_DB_USERNAME:am}
password: ${ROLE_ASSIGNMENT_DB_PASSWORD:am}